### PR TITLE
[IPS-1452] satisfy awesome.re requirements

### PR DESCRIPTION
**Pull request by sukman**
Here is main Awesome repo https://github.com/sindresorhus/awesome

Here are all requirements for contribution to Awesome repo:

- The awesome manifesto https://github.com/sindresorhus/awesome/blob/main/awesome.md
- Creating Your Own List https://github.com/sindresorhus/awesome/blob/main/create-list.md
- Contribution Guidelines https://github.com/sindresorhus/awesome/blob/main/contributing.md
- Requirements for your pull request https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md

Here are some repo examples that were recently added to Awesome repo list:

- https://github.com/georgezouq/awesome-ai-in-finance
- https://github.com/TaptuIT/awesome-devsecops
- https://github.com/SE-ML/awesome-seml
- https://github.com/stingalleman/awesome-audiovisual

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,93 @@
-# Awesome Privacy Papers for Visual Data
-Deep learning papers that address the issue of privacy in visual data.
+# Awesome Privacy Papers for Visual Data [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+The ongoing revolution in analytics and AI requires enormous amount of visual data. At the same time it is social responsibility to respect privacy protection while working with visual data.
+
+We do care deeply about privacy. To expand our knowledge base on this topic we do constant research in scientific community. Here we want to share with you list of deep learning papers related to the topic of privacy in visual data.
```
Comment (ialvizu) [inline README.md:5]:
```suggestion
The ongoing AI driven data analytics revolution requires an enormous amount of visual data. At the same time, it is a social responsibility to protect individuals privacy linked to these data.

We care deeply about privacy.  To strengthen our  knowledge in this field and understand how it relates to visual data, we do constant research in the latest scientific works about this topic.

We want to share with you a curated list of machine/deep learning papers that address topic of privacy in visual data.
```

File: README.md [hunk 2]
```diff
@@ -1,65 +1,93 @@
-# Awesome Privacy Papers for Visual Data
-Deep learning papers that address the issue of privacy in visual data.
+# Awesome Privacy Papers for Visual Data [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
```
Comment (ialvizu) [inline README.md:2]: Add high DPI "privacy" logo in the right with:

`[](https://brighter.ai)`

The link could be to our page. If it is too much "marketing" we can change it to, for example, https://en.wikipedia.org/wiki/Digital_privacy

Comment (ialvizu) [reply README.md:2]: Add a follow us on twitter like they do [here](https://github.com/sindresorhus/awesome-nodejs/blob/master/readme.md).

**Review by ialvizu — CHANGES_REQUESTED** [review]
I compared with the awesome-guidelines and it seems to me that everything is covered. I added suggestions to the main repository description and other two minor things that we can add.

File: README.md (hunk 2 above)
Comment (sukman) [reply README.md:2]: done

File: README.md [hunk 3]
```diff
@@ -1,65 +1,97 @@
...
-Deep learning papers that address the issue of privacy in visual data.
+# Awesome Privacy Papers for Visual Data [![Awesome](https://awesome.re/badge.svg)](https://awesome.re) [![Twitter Follow](https://img.shields.io/twitter/follow/brighterai.svg?style=social)](https://twitter.com/brighterai)
+
+[<img src="logo.png" align="right" width="100">](https://brighter.ai)
+
+The ongoing AI driven data analytics revolution requires an enormous amount of visual data. At the same time, it is a social responsibility to protect individuals privacy linked to these data.
+
+We care deeply about privacy. To strengthen our knowledge in this field and understand how it relates to visual data, we do constant research in the latest scientific works about this topic.
+
+We want to share with you a curated list of machine/deep learning papers that address topic of privacy in visual data.
```
Comment (ialvizu) [inline README.md:9]:
```suggestion
We want to share with you a curated list of machine/deep learning papers that address the topic of privacy in visual data.
```

**Review by ialvizu — APPROVED** [review]
Just one small change in the text.

**Pull request merged**